src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -2127,7 +2113,7 @@ ucs_status_t uct_ib_mlx5_devx_md_open(struct ibv_device *ibv_device,
         goto err_free_context;
     }
 
-    status = uct_ib_mlx5_check_uar(md);
+    status = uct_ib_mlx5_devx_check_supported_uar_alloc_type(md);
```

Comment:
let's keep the previous short function name "uct_ib_mlx5_check_uar"

Reply:
renamed to "uct_ib_mlx5_devx_check_uar"